packages/blazor-workspace/Tests/NimbleBlazor.Tests/Unit/Components/NimbleNumberFieldTests.cs

```diff
@@ -15,7 +16,7 @@ public class NimbleNumberFieldTests
     [Fact]
     public void NimbleNumberField_Rendered_HasNumberFieldMarkup()
     {
-        var context = new TestContext();
+        using var context = new TestContext();
```

Comment:
It would be helpful if the PR description mentioned the rationale for the `using` change.

From some digging, [it appears to match the Bunit docs](https://bunit.dev/docs/getting-started/writing-tests.html?tabs=xunit#instantiate-bunitcontext-in-each-test) (TestContext was renamed to BunitContext) so I think this is a positive change. But if it is then we should apply it to [everywhere that creates a TestContext](https://github.com/search?q=repo%3Ani%2Fnimble%20testcontext&type=code) in a separate PR.